app/Mails/MailJet/MailjetMailer.php

```diff
@@ -23,6 +23,12 @@ public function __construct(
 
     public function send(Address $from, Address $to, int $templateId, array $variables = []): void
     {
+        if (!filter_var($from->getAddress(), FILTER_VALIDATE_EMAIL)) {
```

Comment:
This is again an example of converting a valid runtime error to something that silently continues.

Reply:
This issue should not occur with new integrations due to the updated email validation process. The fix is created for backwards compatibility with old data.

Currently email reminders fail when encountering an integration with invalid data, requiring manual fixes for the email addresses.

One alternative is to silence all errors from the Mailjet API, but this would prevent distinguishing between a failure caused by an invalid email and other API issues. As a result, we wouldn’t be able to confirm if the email was actually sent, only that an attempt was made to call the API.

Reply:
Catch the errors and log them?

Reply:
I removed it